builder/dockerfile/builder_test.go

```diff
@@ -0,0 +1,54 @@
+package dockerfile
+
+import (
+	"strings"
+
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/builder/dockerfile/parser"
+)
+
+func TestBuildProcessLabels(t *testing.T) {
+	dockerfile := "FROM scratch"
+	d := parser.Directive{}
+	parser.SetEscapeToken(parser.DefaultEscapeToken, &d)
+	n, err := parser.Parse(strings.NewReader(dockerfile), &d)
+	if err != nil {
+		t.Fatalf("Error when parsing Dockerfile: %s", err)
+	}
+
+	options := &types.ImageBuildOptions{
+		Labels: map[string]string{
+			"org.a": "cli-a",
```

Comment:
For clarity, yes I agree.

In practice, it won't make a difference, because Go always randomizes map order; see https://nathanleclaire.com/blog/2014/04/27/a-surprising-feature-of-golang-that-colored-me-impressed/